symphony/content/content.publish.php

```diff
@@ -1775,19 +1778,21 @@ private function prepareAssociationsDrawer($section)
                     $header = new XMLElement('header');
 
                     // Get the search value for filters and prepopulate
-                    $entry = current(EntryManager::fetch($entry_id));
-                    $search_value = $relation_field->fetchAssociatedEntrySearchValue(
-                        $entry->getData($as['parent_section_field_id']),
-                        $as['parent_section_field_id'],
-                        $entry_id
-                    );
-                    if (is_array($search_value)) {
-                        $search_value = $entry_id;
+                    $filter      = '';
+                    $prepopulate = '';
+                    if ($entry = current(EntryManager::fetch($entry_id))) {
```

Comment:
Is this construct still encouraged ? I mean, it is not super clear that it is an assignment and not a boolean test.

Reply:
I don’t think that it was ever encouraged. Not sure if it’s specifically discouraged, though. I don't have a strong opinion about it, can change it to separate lines if you find it more readable that way.

Reply:
Yes I think it is more readable on two lines. Thanks.

Reply:
Done.